domain/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 // Create JAR with test fixtures
 task testJar(type: Jar, dependsOn: testClasses) {
-    baseName = 'test-${project.archivesBaseName}'
+    baseName = 'test-' + project.archivesBaseName
```

Comment:
not sure whether a test jar is needed here, but before this change the jar was named `test-${project.archivesBaseName}.jar` as the property wasn't being replaced.

Reply:
@timja - I believe `project.archivesBaseName` is a property added by the `java` plugin (see https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#N1506F), which is applied to all projects via `allprojects` in the parent `build.gradle`.

The syntax itself (i.e. `${project.archivesBaseName}`) should be correct; for example, it's used in the following:
https://discuss.gradle.org/t/how-to-publish-jar-with-test-fixtures-in-gradle/17678
https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/build.gradle

However, I'm not sure if `archivesBaseName` is defaulted to anything if not set explicitly.

Reply:
@Daniel-Lam after my change it is: `./domain/build/libs/test-domain-1.2.0-SNAPSHOT-tests.jar`

before my change it was: `./domain/build/libs/test-${project.archivesBaseName}-tests.jar`

Reply:
@timja - hmm... that's odd. I spotted `test-${project.archivesBaseName}-1.2.0-SNAPSHOT-tests.jar` in my local build, too. So, it looks like `project.archivesBaseName` _does_ have the correct value but, for some reason, the `${...}` syntax doesn't work. Oh well :)